### PR TITLE
Add New Review Form and details to Review Tiles

### PR DESCRIPTION
**Pull request by pdaly91**
Add Pictures and Helpfulness adder to review tiles
https://trello.com/c/k6pHtVom

Form to Add New Review
https://trello.com/c/MIs9k5Jk

### CODE REVIEW

File: src/components/ratingsAndReviews/NewReviewModal.jsx
Comment (andymoch) [inline src/components/ratingsAndReviews/NewReviewModal.jsx:1]: Loving all the switch cases and readability of it.

File: src/apis/reviews.js
Comment (andymoch) [inline src/apis/reviews.js:1]: Great work implementing the http requests getting them to work.

**Review by andymoch — APPROVED** [review]
Fantastic work Patrick

File: src/apis/reviews.js
Comment (pdaly91) [reply src/apis/reviews.js:1]: Thanks! I still need to figure out the PUT requests for review helpfulness, but I'm glad the POST request worked!

**Pull request merged**